### PR TITLE
[Frontend V2] Fix alignment of visibility option on submit page

### LINKED ISSUE #3349: [Frontend V2] Fix alignment of visibility option on submit page
### Description

The `Select submission visibility` option is misaligned on the `Submit` tab of the challenge page. It should be aligned with submission metadata fields. Here's how it looks currently.
Screen Shot 2021-03-24 at 12 43 47 PM

### PR DESCRIPTION
**Pull request by Alabhya268**
fixed #3349

It seems I forget to add css when making #3229.

submit

### CODE REVIEW

**Comment by Alabhya268** [comment]
@Ram81, Please review this PR.

File: frontend_v2/src/app/components/challenge/challengesubmit/challengesubmit.component.scss [hunk 1]
```diff
@@ -122,3 +122,7 @@ th {
 #submission_option_style {
   margin-left: 1em;
 }
+
+.submission_visibility {
+  margin-left: 1em;
```
Comment (Ram81) [inline frontend_v2/src/app/components/challenge/challengesubmit/challengesubmit.component.scss:127]: @Alabhya268 let's not add a new css class for every change. Please put this change in existing css class or use one of the existing css class for this field.

**Review by Ram81 — CHANGES_REQUESTED** [review]
@Alabhya268 the spacing between submission visibility and the next field is inconsitent with submission meta attributes field spacing. Can we fix that too.

**Comment by Alabhya268** [comment]
@Ram81, I've fixed submit page css issue using pre written css class.
same classes are used as submission ```meta attributes```.
This also fixed space issues between ```submission visibility``` and ```submission meta```.
Please review again.

sss

**Comment by Ram81** [comment]
@Alabhya268 can you also share a screenshot of a challenge with additional submission meta attributes.

**Comment by Alabhya268** [comment]
> @Alabhya268 can you also share a screenshot of a challenge with additional submission meta attributes.

@Ram81, There is a bug in frontendv2, it not showing more submission attributes.

Frontentv1 screenshot


Frontendv2 screenshot



As you can see frontendv2 does not show additional submission meta attributes.
This has nothing to do with this PR as I have also tested it on the master branch.

**Comment by Alabhya268** [comment]
@Ram81

File: frontend_v2/src/app/components/challenge/challengesubmit/challengesubmit.component.html [hunk 2]
```diff
@@ -236,7 +236,8 @@ <h5 class="fw-light">Make Submission</h5>
                     </div>
                   </div>
                 </div>
-                <div>
+                <div class="row row-lr-margin">
+                  <div class="align-left col-sm-10 col-lr-pad">
                   <p class="fs-14 w-400">Select submission visibility:</p>
```
Comment (Ram81) [inline frontend_v2/src/app/components/challenge/challengesubmit/challengesubmit.component.html:241]: @Alabhya268 Can you please indent the content of the `<div>` tag.

**Review by Ram81 — CHANGES_REQUESTED** [review]
@Alabhya268 minor change

File: frontend_v2/src/app/components/challenge/challengesubmit/challengesubmit.component.html (hunk 2 above)
Comment (Alabhya268) [reply frontend_v2/src/app/components/challenge/challengesubmit/challengesubmit.component.html]: > @Alabhya268 minor change

@Ram81, I have fixed the indent, Please look at this PR again.

**Comment by Alabhya268** [comment]
@Ram81, I have fixed the indent, Please look at this PR again.

**Review by Ram81 — APPROVED** [review]
LGTM 👍

**Comment by Alabhya268** [comment]
@RishabhJain2018

**Review by RishabhJain2018 — APPROVED** [review]

**Pull request merged**